jenkins/build.sh

```diff
@@ -43,6 +47,6 @@ done
 # exit code.  In this case, if the build command fails, the tee
 # command won't trick Jenkins into thinking the step passed.
 set -o pipefail
-npm --no-color run build -- --buildtype="$envName" --asset-source="$assetSource" --drupal-address="$drupalAddress" "$omitdebug" "$pullDrupal" 2>&1 | tee "$buildLog"
+npm --no-color run build -- --verbose --buildtype="$envName" --asset-source="$assetSource" --drupal-address="$drupalAddress" --drupal-max-parallel-requests="$drupalMaxParallelRequests" "$omitdebug" "$pullDrupal" 2>&1 | tee "$buildLog"
```

Comment:
I added the `verbose` flag here too. It looks like it was intended to always be here but it wasn't passed over to this bash script, https://github.com/department-of-veterans-affairs/vets-website/pull/16275/files#diff-2cc6a42e83abdf9666365745cd63a9a93da477f58972350809e0b2acd32dfaceR215

Reply:
Here is the output added by the verbose flag

![image](https://user-images.githubusercontent.com/1915775/110142800-8b4b2b80-7da4-11eb-8404-e8dfbe99a85f.png)

What's cool is that we are building a site at over 2300 pages now and still using like 1/6 of the memory the build was using back in January.